addon-test-support/@ember/test-helpers/dom/click.ts

```diff
@@ -31,11 +30,15 @@ export const DEFAULT_CLICK_OPTIONS = {
   @param {Element} element the element to click on
   @param {MouseEventInit} options the options to be merged into the mouse events
 */
-export function __click__(element: Element | Document, options: MouseEventInit): void {
+export function __click__(element: Element | Document | Window, options: MouseEventInit): void {
   fireEvent(element, 'mousedown', options);
 
-  if (isFocusable(element)) {
-    __focus__(element);
+  if (!isWindow(element)) {
```

Comment:
Why do we need this? If you widen `isFocusable` and `isFormControl` to allow passing `window` and just return `false` wouldn't that be fine? I think if we do that it may simplify the rest of the code?

Reply:
I thought about that too, but it pulls the thread all the way through those APIs, which also feel odd to widen to accept window.

Reply:
OK as we discussed offline, I've moved to using a type guard on the `isWindow` function, which has negated the need for this.